src/components/Claimer.tsx

```diff
@@ -180,7 +271,7 @@ const Claimer: React.FC<IClaimer> = ({
             <Text fontSize="18px" color="lid.fg">
               {`More ${meta.tokenSymbol} available to claim in`}
             </Text>
-            <CountDownShort expiryTimestamp={claimPeriod * 1000} />

```

Comment:
Did you test this line? The expiryTimestamp should be in milliseconds, I think.

Reply:
I did not add that line.

Reply:
Can double check though

Reply:
but I can see this line is changed